packages/arb-token-bridge-ui/tests/e2e/specs/depositETH.cy.ts

```diff
@@ -10,96 +10,63 @@ describe('Deposit ETH', () => {
-            cy.findByText('Total amount')
-              .siblings()
-              .last()
-              .contains(/(\d*)(\.\d+)*( ETH)/)
-              .should('be.visible')
-          })
-      })
-
-      it('should deposit successfully', () => {
-        cy.findByRole('button', {
-          name: 'Move funds to Arbitrum'
-        }).click({ scrollBehavior: false })
-        // https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Scrolling
-        // cypress by default tries to scroll the element into view even when it is already in view
-        // for unknown reasons, probably due to our root div's overflow:hidden CSS property,
-        // cypress would wrongly scroll the div and bring the element to the top of the view
-        // and in turn include the full moon into the view, cropping the header out of visible area
-        // to circumvent this bug with cypress, scrollBehaviour should be set false for this element
-        // because the element is already in view and does not require scrolling
-        // https://github.com/cypress-io/cypress/issues/23898
+  it('should show L1 and L2 chains correctly', () => {
+    cy.findByRole('button', { name: /From: Ethereum/i }).should('be.visible')
+    cy.findByRole('button', { name: /To: Arbitrum/i }).should('be.visible')
+  })
 
-        cy.confirmMetamaskTransaction().then(() => {
-          cy.findByText(
-            `Moving ${formatAmount(0.0001, {
-              symbol: 'ETH'
-            })} to Arbitrum`
-          ).should('be.visible')
-        })
+  it('should show summary and bridge successfully', () => {
+    cy.findByPlaceholderText('Enter amount')
+      .typeRecursively(String(ETHAmountToDeposit))
+      .then(() => {
+        cy.findByText('You’re moving')
+          .siblings()
+          .last()
+          .contains(formatAmount(0.0001, { symbol: 'ETH' }))
+          .should('be.visible')
+        cy.findByText('You’ll pay in gas fees')
+          .siblings()
+          .last()
+          .contains(zeroToLessThanOneETH)
+          .should('be.visible')
+        cy.findByText('L1 gas')
+          .parent()
+          .siblings()
+          .last()
+          .contains(zeroToLessThanOneETH)
+          .should('be.visible')
+        cy.findByText('L2 gas')
+          .parent()
+          .siblings()
+          .last()
+          .contains(zeroToLessThanOneETH)
+          .should('be.visible')
+        cy.findByText('Total amount')
+          .siblings()
+          .last()
+          .contains(/(\d*)(\.\d+)*( ETH)/)
+          .should('be.visible')
       })
+    cy.findByRole('button', {
+      name: 'Move funds to Arbitrum'
+    }).click({ scrollBehavior: false })
+    cy.confirmMetamaskTransaction().then(() => {
+      cy.findByText(
+        `Moving ${formatAmount(0.0001, {
+          symbol: 'ETH'
+        })} to Arbitrum`
+      ).should('be.visible')
     })
+  })
 
+  context("bridge amount is lower than user's L1 ETH balance value", () => {
```

Comment:
I don't think this context should be here. It was wrapping all the existing tests earlier.

Reply:
done